src/components/Footer.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+export default function Footer(props) {
+    const li = props.data.map((data, index) => <li key={index}>{data}</li>)
```

Comment:
I actually quite like following the pattern you've got here, but most people who write React would normally *inline* the `li` variable within the curly braces.

I think sometimes it's useful to separate them out like you have though.